src/Illuminate/Support/ConfigurationUrlParser.php

```diff
@@ -1,8 +1,7 @@
 <?php
 
-namespace Illuminate\Database;
+namespace Illuminate\Support;
```

Comment:
Unfortunately this is a breaking chance since the class was already released. We should keep this in the current component and move it in the next release is that's wanted.

Reply:
I can also see this is problematic with the current PR because it would mean that the `illuminate/database` component would need to be required for the `illuminate/redis` component. Not sure what the best way is to solve this.

We could decide to wing it and just move the class in the current release. Think that the best option for now but unfortunately it'll be a breaking change. Otherwise the only thing we can do is do the changes in the next release.

Reply:
Indeed, our app is crashing since this changes....

Reply:
@tim-hanssen is it because you were using the class directly? What do you mean with "crashing"? Please provide some more info.

Reply:
We're using predis (reddis running as a cluster).

![image](https://user-images.githubusercontent.com/11192051/58561690-f5c25580-8227-11e9-88ae-6cd498905bde.png)

Since version 5.8.19 it crashes.

Reply:
@driesvints The "breaking change" problem has been resolved immediately by Taylor, before releasing anything, so this is resolved.

However @tim-hanssen you seem to have a real issue, so could you open it as an new issue and ping me in it please?

Thanks.